Source/ACE/Entity/Landblock.cs

```diff
@@ -359,14 +372,20 @@ public void UseTime()
                 // for now, we'll move players around
                 List<WorldObject> movedObjects = null;
                 List<Player> players = null;
+                List<Container> despawnObjects = null;
+                List<Creature> deadCreatures = null;
 
                 lock (objectCacheLocker)
                 {
                     movedObjects = this.worldObjects.Values.OfType<WorldObject>().ToList();
                     players = this.worldObjects.Values.OfType<Player>().ToList();
+                    despawnObjects = this.worldObjects.Values.OfType<Container>().ToList();

```

Comment:
I would think that we would want to be able to despawn anything.  Case in point, junk loot you drop on the ground needs to go away eventually.  Perhaps use WorldObject instead of Container?